weldx/config.py

```diff
@@ -171,6 +171,14 @@ def enable_quality_standard(name: str, version: Union[AsdfVersion, str] = None):
         asdf_config.add_resource_mapping(manifest_mapping)
         asdf_config.add_resource_mapping(schema_mapping)
 
+        # Since we modified the resource mapping, we have to clear the already loaded
+        # schema cache to make it possible to set another schema for a type.
+        # Note that this could bear inefficiencies due to lack of caching!
+        # FIXME: this is a hack into impl details of asdf and should be changed.
+        from asdf.schema import _load_schema_cached
+
+        _load_schema_cached.cache_clear()
+
```

Comment:
I removed the changes because the loaded quality standard is now kept active for some reason. This causes validation errors in the follow-up tests. `wx_metadata` is now a required property in all other test files. So the behavior gets somehow worse. I think we should tackle this issue in another PR so that it doesn't prevent us from merging this PR.